doc/exts/pylint_messages.py

```diff
@@ -29,6 +31,8 @@
 
 MSG_TYPES_DOC = {k: v if v != "info" else "information" for k, v in MSG_TYPES.items()}
 
+BASE_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I think we can use PYLINT_BASE_PATH instead.

Reply:
```suggestion
PYLINT_BASE_PATH = Path(_file__).parent.parent.resolve()
```